### ISSUE TITLE
call_user_func_array() expects parameter 1 to be a valid callback, cannot access private method UA_Made_Rewrite_Rules::view()

### ISSUE THREAD

**Issue by byteart** [butuzov/Debug-Bar-Rewrite-Rules#2]
Debug Bar Rewrite Rules Version 0.5

rewrite-rules.php on line 216
```php
	private function view() {
		echo // WPCS: XSS OK.
			'<div class="wrap debug-bar-rewrites-urls">',
			sprintf( '<h2>%s</h2>', $this->pagetitle ),
			$this->stats(),
			$this->rules(),
			$this->filters(),
			'</div>';
	}
```

**Comment by butuzov** [comment]
valid point. php version?

**Comment by byteart** [comment]
PHP Version 5.6.30

**Comment by butuzov** [comment]
FIxed in d955c58. You can use code from dev branch, or wait a bit and download it on wordpress.org (guess its time to add more bugs/features and deploy new version).
